package.json

```diff
@@ -27,7 +27,7 @@
     "packages/angular/projects/ui-angular"
   ],
   "resolutions": {
-    "@mdx-js/mdx/**/trim": "^0.0.3",
```

Comment:
I checked in `node_modules` and this wasn't forcing the version change. `node_modules/trim/package.json` still showed a version of `0.0.1`. Updating to just the module name changed the `trim` version to `0.0.3`

Reply:
Thanks for this fix!